src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -17,7 +17,7 @@ private static partial class Settings
 #else
             internal static bool Invariant { get; } = AppContextConfigHelper.GetBooleanConfig("System.Globalization.Invariant", "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
 #endif
-#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS || TARGET_BROWSER
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS || TARGET_BROWSER || TARGET_WASI
```

Comment:
Why does WASI needs this defined? Hybrid globalization does not make sense on WASI.

Reply:
Whatever the reason was it's gone now.